packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts

```diff
@@ -141,7 +141,7 @@ describe('Telemetry', () => {
       const telemetryEnabled = await telemetryService.isTelemetryEnabled();
       expect(telemetryEnabled).to.be.eql(true);
 
-      showTelemetryMessage(mockExtensionContext);
+      await showTelemetryMessage(mockExtensionContext);
```

Comment:
Can you make the same change to all 4 occurrences of `showTelemetryMessage()` in this file?